### DISCUSSION TITLE
Ensemble evaluator MONAI

### DISCUSSION THREAD

**Discussion by cugwu** [Project-MONAI/MONAI#4806] [Q&A]
I've implemented the tutorial in:
https://github.com/Project-MONAI/tutorials/blob/main/modules/cross_validation_models_ensemble.ipynb
related to the ensemble evaluator. 
I wondered, how can I visualize the final predictions obtained with the mean dice or majority voting? Is there a way to have the final model used inside the ensemble evaluator to obtain the final results? I just need the predictions in order to have qualitative results.

My second question is, how can I remove the following message "INFO:nibabel.global:pixdim[0] (qfac) should be 1 (default) or -1; setting qfac to 1"? Is not a simple warning so I cannot remove it by just suppressing all the warnings.

**Comment by wyli** [comment]
for the 2nd one I think you can try `import nibabel; nibabel.imageglobals.logger.setLevel(40)`

**Reply by cugwu** [reply]
Thanks very much it works! What level 40 means?

**Comment by Nic-Ma** [comment]
Hi @cugwu ,

MONAI supports many different visualization methods, you can refer to the tutorial:
https://github.com/Project-MONAI/tutorials/blob/main/modules/transform_visualization.ipynb

Thanks.

**Reply by cugwu** [reply]
Hi, 
My problem is not visualizing the images but accessing the final predictions of "ensemble_evaluate" (implementation at: https://github.com/Project-MONAI/tutorials/blob/main/modules/cross_validation_models_ensemble.ipynb), in order to then apply one of the visualization methods.
Thanks in advance for the answer

**Reply by KumoLiu** [reply]
Hi @cugwu,
You can simplely add a `SaveImaged` in the end of your post transform.

```
from monai.transforms import SaveImaged
mean_post_transforms = Compose([
......
SaveImaged(keys="pred", output_dir="./out", output_postfix="seg", resample=False)
])

```
